website/docs/docs/dbt-cloud/access-control/self-service-permissions.md

```diff
@@ -18,3 +16,20 @@ dbt Cloud supports two different permission sets to manage permissions for self-
 | Manage billing | ❌ | ✅ |
 | Manage team permissions | ❌ | ✅ |
 | Invite Owners to the account | ❌ | ✅ |
+
+## Member and Owner Groups in dbt Cloud Enterprise
+
+By default, new users are added to the Member and Owner groups when they onboard to dbt Cloud. This is necessary for account onboarding but can create confusion when initially setting up SSO and RBAC for dbt Cloud Enterprise accounts as described in the [Enterprise Permissions](https://docs.getdbt.com/docs/dbt-cloud/access-control/enterprise-permissions/) guide. Users may be added with more permissions than intended without adjusting this setting.
+
+We recommend the following steps be taken after onboarding Administrative users and before adding less priveleged users to a dbt Cloud Enterprise account.
+
+1) Click the Hamburger icon at the top left of the Cloud Console, then Account Settings >> Groups >> <Owner/Member>  
+2) Uncheck the "ADD USERS BY DEFAULT" option:  
+
+<Lightbox src="static/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/uncheck-add-users-by-default.png" title="Adjusting default group assignments"/>
+
+With this setting disabled, permission sets will apply transparently to additional groups configured in dbt Cloud.
+
+## Read Only vs. Developer License Types
```

Comment:
I personally think this section should come up higher, before the enterprise section.

Reply:
Cool - swapped these so ##Read Only comes first.

Reply:
Moved.